cinema/views.py

```diff
@@ -116,14 +139,20 @@ def get_serializer_class(self):
 class OrderPagination(PageNumberPagination):
     page_size = 10
     max_page_size = 100
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
```

Comment:
The `OrderPagination` class should not have `authentication_classes` and `permission_classes` attributes. These are not applicable to pagination classes and should be removed to avoid confusion or potential errors.